src/test/java/nl/tudelft/pixelperfect/gamestates/PlayStateTest.java

```diff
@@ -60,7 +62,7 @@ public void testUpdate() {
     verify(mockScheduler).update(1f);
     verify(mockSpaceship).update(anyFloat());
 
-    if (Constants.isDebug) {
+    if (Settings.getIsDebug()) {
       verify(mockHeadsUp).updateHud();
     }
```

Comment:
I totally agree. David, what are you even verifying here, and why? Can't you create two cases, one where you set Settings to debug mode and one where you do not?